hasura/metadata/actions.yaml

```diff
@@ -112,6 +112,15 @@ actions:
       value_from_env: HASURA_EVENT_SECRET
   permissions:
   - role: user
+- name: restoreCoordinape
+  definition:
+    kind: synchronous
+    handler: '{{HASURA_API_BASE_URL}}/actions/restoreCoordinape'
```

Comment:
we recently changed the way we add handlers (see #1046 for context); please follow the new pattern